components/FormComponents.js

```diff
@@ -2,7 +2,24 @@ import styled from "styled-components";
 import { IconChevronDown, IconChevronUp, IconList, IconPlus } from "./Icons";
 
 import { PTag } from "./HtmlComponents";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+
+const FormMain = styled.main`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  min-height: calc(100vh - 81px);
+  padding: 0 20px;
+`;
+
+const StyledForm = styled.form`
+  margin-top: -81px;
+  display: flex;
+  flex-flow: column;
+  gap: 10px;
+  width: 100%;
```

Comment:
same here, se above

Reply:
@F-Kirchhoff no. not here. cause the above component (FormMain) is flexed